_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -62817,6 +62833,10 @@
 /obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden/layer4,
 /turf/open/floor/iron,
 /area/hallway/secondary/command)
+"wip" = (
+/obj/machinery/door/poddoor,
+/turf/closed/wall/r_wall,
+/area/cargo/miningdock)
```

Comment:
Why is there a pod door on the same tile as an r-wall in the mining dock area? there should be no rwalls in this area anyway?

Reply:
![image](https://user-images.githubusercontent.com/85910816/133660728-c10bda67-5f49-43c9-b3ea-086786053ea3.png)
I guess I accidentally placed one here while I was trying to figure out the program